src/lib/setup/azureContainerRegistryService.ts

```diff
@@ -0,0 +1,110 @@
+import { ContainerRegistryManagementClient } from "@azure/arm-containerregistry";
```

Comment:
I would suggest moving this file to `lib\azure` folder since it supposed to have all `.ts` files related to `azure` .

Reply:
accept